src/asset.js

```diff
@@ -33,7 +33,7 @@ class Asset extends Events {
             path: []
         }, data);
 
-        this._observer = new Observer(data, options);
+        this._observer = new Observer(data, options || {});
```

Comment:
If `options` is undefined, the  constructor should create one as a default parameter, no?

https://github.com/playcanvas/playcanvas-observer/blob/main/src/observer.js#L13

Reply:
Yes. Though here options is initialised to `null`.

Reply:
OK, sure, but why not just leave it uninitialized then?